src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_IsByRefLike.cs

```diff
@@ -29,11 +31,17 @@ class Test
 }
 ";
 
-            CompileAndVerify(text, verify: Verification.Passes, symbolValidator: module =>
+            void validate(ModuleSymbol module)
             {
                 var type = module.ContainingAssembly.GetTypeByMetadataName("Test").GetTypeMember("S1");
-                AssertReferencedIsByRefLikeAttributes(Accessibility.Public, type, module.ContainingAssembly.Name);
-            });
+                AssertReferencedIsByRefLike(type);
+
+                var peModule = (PEModuleSymbol)module;
+                Assert.True(peModule.Module.HasIsByRefLikeAttribute(((PENamedTypeSymbol)type).Handle));
+                Assert.Equal(Accessibility.Public, peModule.GetTypeSymbolForWellKnownType(WellKnownType.System_Runtime_CompilerServices_IsByRefLikeAttribute).DeclaredAccessibility);
```

Comment:
>GetTypeSymbolForWellKnownType [](start = 60, length = 29)

I just realized that this helper is probably not quite what we should use here. In particular it does not guarantee that the type it finds is defined in this module. It feels like an existing test helper that looks for a type in the given container (module's global namespace in our case) is a better fit. I think there is one that takes a fully qualified name.

Reply:
Thanks
Narrowed the check. I couldn't find an existing helper, so followed the pattern used a bunch of NoPia tests (GetMetadataName and use it in Lookup method).

---
In reply to: [168557695](https://github.com/dotnet/roslyn/pull/24492#discussion_r168557695) [](ancestors = 168557695)

Reply:
>  I couldn't find an existing helper

src\Compilers\Test\Utilities\CSharp\Extensions.cs
```
public static T GetMember<T>(this NamespaceOrTypeSymbol symbol, string qualifiedName) where T : Symbol
        {
            return (T)symbol.GetMember(qualifiedName);
        }
```

---
In reply to: [168574351](https://github.com/dotnet/roslyn/pull/24492#discussion_r168574351) [](ancestors = 168574351,168557695)